src/common/check-usage.js

```diff
@@ -8,7 +8,9 @@ const checkUsage = function (manifest, varName) {
   for (let i = 0; i < containers.length; i++) {
     const container = containers[i]
     const envVars = container['environment']
-
+    if (!envVars) {
```

Comment:
I would slightly prefer this:

```js
const envVars = container['environment'] || {}
```

If somebody changes this section in the future, they could easily miss the `continue`. It's good to minimize the number of different paths there are through your code.